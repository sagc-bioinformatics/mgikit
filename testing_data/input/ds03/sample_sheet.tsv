sample_id	i7	i5
SAM-01	CGTGACATGC	ATGGTCTAAA
SAM-02	CGGAACCCAA	GATTCGAGGA
SAM-03	TGGTGCGGTG	GACTGTCAAT
SAM-04	GATTTCAGCG	GCAGACACCT
