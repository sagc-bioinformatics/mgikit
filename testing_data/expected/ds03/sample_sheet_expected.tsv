sample_id	i7	i5	job_number	template	i7_rc	i5_rc
SAM-01	CGTGACATGC	ATGGTCTAAA	.	i710:i510	0	0
SAM-02	CGGAACCCAA	GATTCGAGGA	.	i710:i510	0	0
SAM-03	TGGTGCGGTG	GACTGTCAAT	.	i510:i710	1	0
SAM-04	GATTTCAGCG	GCAGACACCT	.	i710:i510	1	1
